### PR TITLE
Package pilat.1.3

### PR DESCRIPTION
**Pull request by Stevendeo**
### `pilat.1.3`
A polynomial invariant synthesizer

---
* Homepage: https://github.com/Stevendeo/Pilat/
* Source repo: git+https://github.com/Stevendeo/Pilat.git
* Bug tracker: https://github.com/Stevendeo/Pilat/

---
:camel: Pull-request generated by opam-publish v2.0.2

### CODE REVIEW

**Comment by kit-ty-kate** [comment]
No archive has been defined, the tag for the version 1.3 is missing from your repository. Could you push the tag and resubmit using opam-publish? (no need to close this PR, `opam publish` will automatically update it).

Sorry for the delay

File: packages/pilat/pilat.1.3/opam [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+bug-reports: "https://github.com/Stevendeo/Pilat/"
+license: "LGPLv2.1"
+build: ["sh" "make.sh"]
+install: ["sh" "install.sh"]
+remove: ["ocamlfind" "remove" "pilat"]
+depends: [
+  "ocaml" {> "4.02.3"}
+  "ocamlfind"
+  "lacaml"
+  "zarith"
```
Comment (mseri) [inline packages/pilat/pilat.1.3/opam]: Do you know if this requires additional lower bounds?

File: packages/pilat/pilat.1.3/opam [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+dev-repo: "git+https://github.com/Stevendeo/Pilat.git"
+homepage: "https://github.com/Stevendeo/Pilat/"
+bug-reports: "https://github.com/Stevendeo/Pilat/"
+license: "LGPLv2.1"
+build: ["sh" "make.sh"]
+install: ["sh" "install.sh"]
+remove: ["ocamlfind" "remove" "pilat"]
+depends: [
+  "ocaml" {> "4.02.3"}
+  "ocamlfind"
```
Comment (mseri) [inline packages/pilat/pilat.1.3/opam]: I only had a very brief look but I think this is just a build dependency
```suggestion
  "ocamlfind" {build}
```

**Comment by kit-ty-kate** [comment]
Thanks! Could you return the fixes upstream?

**Pull request merged**